lib/nitcorn/examples/src/nitcorn_reverse_proxy.nit

```diff
@@ -0,0 +1,9 @@
+import nitcorn
+
+var vh = new VirtualHost("localhost:8080")
+var proxy_interface = new Interface("localhost", 31337)
+vh.routes.add new Route("/", new ProxyAction(proxy_interface))
+
```

Comment:
Maybe we should add the NIT_TESTING thing in nitcorn itself as it is done with mnit https://github.com/nitlang/nit/blob/master/lib/mnit/mnit_app.nit#L66